### PR TITLE
docs: add `catalogsMergePath` in Lingui Configuration

### LINKED ISSUE #1879: The `catalogsMergePath` option is missing in the documentation
[Related code](https://github.com/search?q=repo%3Alingui%2Fjs-lingui+catalogsMergePath&type=code)

This option needs to be described in the [Lingui Configuration](https://lingui.dev/ref/conf) section.

### PR DESCRIPTION
**Pull request by aseerkt**
# Description

Include `catalogsMergePath` in [Lingui Configuration](https://lingui.dev/ref/conf) reference section in docs

## Types of changes

[//]: # (What types of changes does your code introduce to Lingui?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Examples update

Fixes #1879 

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/lingui/js-lingui/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT](https://github.com/lingui/js-lingui/blob/main/CODE_OF_CONDUCT.md) docs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation (if appropriate)

### CODE REVIEW

File: website/docs/ref/conf.md [hunk 1]
```diff
@@ -180,6 +181,28 @@ components/
         └── LoginForm.js
 ```
 
+## catalogsMergePath
+
+Default: `""`
+
+Specify the path to merge translated catalogs into a single file per locale during compile
```
Comment (andrii-bodnar) [inline website/docs/ref/conf.md:188]:
```suggestion
Specify the path to merge translated catalogs into a single file per locale during compile.
```

File: website/docs/ref/conf.md [hunk 2]
```diff
@@ -180,6 +181,28 @@ components/
...
+
+#### Example
+
+```json
+{
+  "catalogsMergePath": "locales/{locale}"
+}
+```
+
+The catalogs will be compiled to `locales/en.js` and `locales/cs.js`
```
Comment (andrii-bodnar) [inline website/docs/ref/conf.md:198]:
```suggestion
For `locales: ["en", "cs"]` the catalogs will be compiled into `locales/en.js` and `locales/cs.js`:
```

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

**Comment by thekip** [comment]
BTW it still not clear from the docs what actually it does. Maybe add some examples with file tree before and after?

**Comment by andrii-bodnar** [comment]
@aseerkt Thanks! I would suggest making the example more compact: use the code `diff` highlight to show what the file structure is before the merge (-) and after the merge (+) on a single block of code.

**Comment by aseerkt** [comment]
Thanks for the suggestion.
PTAL @andrii-bodnar

File: website/docs/ref/conf.md [hunk 3]
```diff
@@ -180,6 +181,55 @@ components/
...
+
+Default: `""`
+
+Specify the path to merge translated catalogs into a single file per locale during compile.
+
+#### Example
+
+Let's assume we have [separate catalogs for `locales: ["en", "cs"]` per component placed inside shared directory](#separate-catalogs-per-component-placed-inside-shared-directory).
+
+Using `catalogsMergePath`, separate catalogs can be merged during compile:
```
Comment (andrii-bodnar) [inline website/docs/ref/conf.md:194]:
```suggestion
Using `catalogsMergePath`, separate catalogs can be merged during [`compile`](/docs/ref/cli.md#compile):
```

**Comment by thekip** [comment]
maybe weird question, but, anyone tested this "Separate catalogs per component, placed inside component dir"? i don't remember this working from the code perspective..

**Comment by aseerkt** [comment]
It seems working @thekip . https://stackblitz.com/edit/vitejs-vite-rkaqac?file=lingui.config.ts

**Comment by aseerkt** [comment]
We need to fix the `catalogs` configuration in the docs in couple of places, where `include` is typed as `string[]`
```diff
{
  "catalogs": [
    {
      "path": "/locale/{locale}/{name}",
-     "include": "components/{name}/"
+     "include": ["components/{name}/"]
    }
  ],
 }
```

**Comment by andrii-bodnar** [comment]
@aseerkt could you please fix this within the current PR?

**Review by andrii-bodnar — APPROVED** [review]
@aseerkt thank you!

**Pull request merged**